Fix to upload_response.html.erb

benrfairless opened the pull request:
Modified to fix a typo, however as this file is not specifically part of our theme I have made it one until the bug is fixed upstream (Upstream issue: https://github.com/mysociety/alaveteli/pull/1510)

benrfairless committed:
Typo Update

Modified to fix a typo, however as this file is not specifically part
of our theme I have made it one until the bug is fixed.

henare commented:
I think you should add a comment to the top of the file noting it can be removed when upstream incorporates the change.

benrfairless commented:
Fair call, I'll have to do this when I get home (so not till late tonight)

benrfairless committed:
Update to upload_response.html.erb

Updated to reflect temporary file which can be removed in 0.19

benrfairless committed:
updated comments so they wouldn't show

Comments added to reflect what the file is for, updated so they
wouldn’t appear in HTML output

henare commented:
Staging deployed.
